ml/backend/ggml/ggml.go

```diff
@@ -257,6 +257,12 @@ func New(modelPath string, params ml.BackendParams) (ml.Backend, error) {
 			if tt := C.ggml_get_tensor(ctxs[bt], cname); tt != nil {
 				return tt
 			}
+			if t.source.Kind == 4 {
+				// HACK: transition from original mxfp4 implementation to ggml implementation
+				t.source.Kind = 39
+			} else if t.source.Kind == uint32(fsggml.TensorTypeBF16) && strings.HasSuffix(t.source.Name, "_exps.bias") {
```

Comment:
Do we need additional checks here to avoid misfiring on other models?  (model arch maybe?)